pkg/discovery/discovery_test.go

```diff
@@ -3,92 +3,100 @@ package discovery
 import (
 	"testing"
 
-	"github.com/stretchr/testify/assert"
+	. "github.com/go-check/check"
```

Comment:
I will convert the `.` imports.

I do not follow you on the `pkg/checker` part. What would that import look like? Is it preferred?